tests/unit/cython/types_testhelper.pyx

```diff
@@ -52,11 +52,11 @@ def test_datetype(assert_equal):
     # deserialize
     # epoc
     expected = 0
-    assert_equal(deserialize(expected), datetime.datetime.utcfromtimestamp(expected))
+    assert_equal(deserialize(expected), datetime.datetime.fromtimestamp(expected, tz=datetime.timezone.utc))
```

Comment:
This one took some digging to sort out but I believe I have it now.  deserialize() returns a naive datetime based on the UTC values so you're correct to continue to resolve the datetime against UTC... but we shouldn't have the timezone in place for what we return.  As a result this needs to become `assert_equal(deserialize(expected), datetime.datetime.fromtimestamp(expected, tz=datetime.timezone.utc).replace(tzinfo=None))`

Reply:
yeah, i missed it in previous fixes.